TreeDB/docs/spec/storage-format.md

```
@@ -489,6 +495,12 @@ values   declared column values
 M12C and later decoders may read version 1 as `Deleted=false` for pre-v2 assets.
 Writers emit version 2.
 
+Sectioned typed-column part payloads are `TreeDB/internal/typedcolumn` part
+images referenced by `ColumnAssetRef.Kind = tcs1_typed_column_part`. The durable
+#1755 scalar path represents bool, int64, float32, double/float64, and string
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)